docs/testplan/pfc/PFC_PAUSE_RESPONSE_HEADROOM_README.md

```diff
@@ -49,4 +49,6 @@ This experiment needs the following five steps:
 
 - And also check if the total number of transmitted bytes of test data traffic is (2) greater than the switch shared buffer size for higher pfc response delay values since the headroom limit would be exceeded. The exact values for the pfc delay response times is dependent on the switch.
 
+- Lastly, check if (1) the number of dropped packets on the egress port (for the specific priority(s)) equals zero when the PFC headroom is not exceeded and (2) is greater than zero when the PFC headroom is exceeded.
```

Comment:
it should be ingress port rather than egress port. Headroom is associated with ingress port.

Reply:
Sorry I meant ingress  - the cisco platform has a centralised egress pool size that is used for both ingress and egress lossless traffic

Reply:
> Sorry I meant ingress - the cisco platform has a centralised egress pool size that is used for both ingress and egress lossless traffic

This has nothing to do with switch ASIC platforms. Packet drops due to PFC headroom blown should be associated with ingress rather than egress.

Reply:
@developfast  This issue has not been resolved. I still see "egress port" on the document.

Reply:
Done resolved.